### ISSUE TITLE
Example9 refers to non exsitant chart option

### ISSUE THREAD

**Issue by magusnebula** [EvotecIT/PSWriteHTML#27]
>  New-HTMLChart -Data @(400, 430, 448) -DataCategories 'Poland', 'Europe', 'Germany' -Type 'donut'

-Data and -Type are not in New-HTMLChart. Maybe old example of where it used to be like this?

**Comment by codaamok** [comment]
Yes I have noticed this too. I believe it has been removed, I no longer see the `-Type` parameter in https://github.com/EvotecIT/PSWriteHTML/blob/master/Public/New-HTMLChart.ps1

**Comment by PrzemyslawKlys** [comment]
Yes, it was removed, as I didn't like the way it was working. I've reimplemented New-HTMLChart in a different way, however, there are still some things that were not ported from old implementation. 

This example needs changes to accommodate the new design, that's available in Example-23. 

I still have some JS code ready for other chart types. 

![image](https://user-images.githubusercontent.com/15063294/60905309-17761a00-a275-11e9-84bf-f11917b22e22.png)

Actually, the plan is to have all of those:

![image](https://user-images.githubusercontent.com/15063294/60905679-d599a380-a275-11e9-9762-378003f74467.png)
